client/landing/stepper/declarative-flow/internals/analytics/record-design.ts

```diff
@@ -90,3 +91,29 @@ export function getVirtualDesignProps( design: Design ) {
 		slug: design.is_virtual ? design.recipe?.slug : design.slug,
 	};
 }
+
+/**
+ * Tracks prop
+ *  name: assembler_source
+ *  values:
+ *		• virtual-theme
+ *		• blank-canvas-theme
+ * 		• standard
+ * 		• premium
+ * 		• default
+ */
+export function getAssemblerSource( design?: Design ) {
```

Comment:
`design` needs to be optional because the test `should call submit successfully` fails when `design` is undefined because users skip the design picker.